src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -78,11 +80,13 @@ static void disableIllegalReflectiveAccessWarning() {
           unsafeClass.getDeclaredMethod(
               "putObjectVolatile", Object.class, long.class, Object.class);
       Method staticFieldOffset = unsafeClass.getDeclaredMethod("staticFieldOffset", Field.class);
+      Method staticFieldBase = unsafeClass.getDeclaredMethod("staticFieldBase", Field.class);
```

Comment:
This is the new (correct) way to get the second input parameter for `pubObjectVolatile` function.